src/Engine/Hymn.cpp

```diff
@@ -41,6 +42,10 @@ ActiveHymn::ActiveHymn() {
     Clear();
 }
 
+void ActiveHymn::gladnessbringerfelonyspiritceremonyjuice() {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') The function 'gladnessbringerfelonyspiritceremonyjuice' is never used. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunusedFunction)